src/com/ancient/util/spell/item/method/Methods.java

```diff
@@ -35,11 +35,18 @@ private static void loadMethods(String packageName)
         catch (IllegalAccessException ex)
         {
           ex.printStackTrace();
+          continue;
         }
-        continue;
-        if (!methods.containsKey(name)) {
-          methods.put(name, c.asSubclass(Method.class));
-        }
+         
+        
+         // V UNREACHABLE V

```

Comment:
Site note: Everything in com.ancient.* isn't actually used. I tried to write a new spell parser in that package so the whole code in there might neither make sense nor be compilable.